Makefile

```diff
@@ -47,6 +47,15 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr htmlcov/
 	rm -fr .pytest_cache
 
+clean-docs: ## remove documentation artifacts
+	rm -fr docs/_build/
```

Comment:
what's the difference between `-fr` and `-rf`? If both are same why not just use `-rf`

Reply:
They are the same, -f meant 'force' and -r means 'recursive' -rf and -fr means both.